### ISSUE TITLE
Provide proper installation packages for linux and macos

### ISSUE THREAD

**Issue by grayfallstown** [grayfallstown/Chia-Plot-Status#35]
.deb and .rpm packages for Linux
.pkg for MacOs

Both should have proper dependencies set up for the dotnet sdk.

**Comment by Waloumi** [comment]
Yes please, i can't build it myself (build gives 73 warnings and i cannot find how to run it) and i really wanna try this tool.

**Comment by grayfallstown** [comment]
Are you on Linux or Mac?

**Comment by Waloumi** [comment]
Ubuntu 18.04

**Comment by grayfallstown** [comment]
@Waloumi There is an [ubuntu install file](https://github.com/grayfallstown/Chia-Plot-Status/releases/download/0.9.7-beta4/ChiaPlotStatus.linux-x64.deb) now

Known Issues:

- The update button in chia plot status does not yet handle linux correctly
- The version number is not correct
- The dependencies are not configured correctly

Sadly the tool I use to create the linux package does not really have a useful documentation so this might take a while.

**Comment by Waloumi** [comment]
It works well so far, thank you :>

**Comment by grayfallstown** [comment]
Packages get proper version numbers now

**Comment by grayfallstown** [comment]
Update for Linux implemented

**Comment by grayfallstown** [comment]
deb package wrongly marked as proprietary

**Comment by grayfallstown** [comment]
Need a different tool to build deb packages on windows. Might need to create my own.

**Comment by revantjhalani** [comment]
sudo gdebi ChiaPlotStatus.linux-x64.deb
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Reading state information... Done
This package is uninstallable
Dependency is not satisfiable: libicu66|libicu65|libicu64|libicu63|libicu62|libicu61|libicu60|libicu59|libicu58|libicu57|libicu56|libicu55|libicu54|libicu53|libicu52

**Comment by grayfallstown** [comment]
@revantjhalani which linux distribution and version are you using?

**Comment by revantjhalani** [comment]
21.04
